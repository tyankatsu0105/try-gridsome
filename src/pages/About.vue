<template>
  <Layout> <h1>About us</h1> </Layout>
</template>

<script>
export default {};
</script>
