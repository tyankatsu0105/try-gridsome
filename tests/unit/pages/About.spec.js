import { shallowMount } from "@vue/test-utils";
import About from "@/pages/About.vue";

describe("About.vue", () => {
  it("render", () => {
    const wrapper = shallowMount(About);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
