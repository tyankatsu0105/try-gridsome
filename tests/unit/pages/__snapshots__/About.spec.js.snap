// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.vue render 1`] = `
<layout>
  <h1>About us</h1>
</layout>
`;
